dhnx/input_output.py

```diff
@@ -401,7 +401,7 @@ def load(self):
 
         footprints = self.download_footprints()
 
-        footprints = ox.project_gdf(footprints)
+        # footprints = ox.project_gdf(footprints)
```

Comment:
Are the footprints already projected in line 215? What I don't understand, what is the original crs, and what the projected?